### PR TITLE
Remove console.warn from code to avoid chattiness when being used

### PR DESCRIPTION
**Pull request by barlock**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
+#node
 node_modules/*
+
+#intelij
+.idea
```
Comment (jagoda) [inline .gitignore]: What are these changes needed for? Seems like clutter to me..

**Comment by jagoda** [comment]
Looks fine other than the ignore file options that seem a little environment specific.

**Comment by barlock** [comment]
Updated .gitignore per your comments
